### PR TITLE
Update omero import tools

### PR DESCRIPTION
**Pull request by lldelisle**
This PR contains 2 changes:
- Do not expose password in command line when possible
- Use label for ROI in omero for all shapes as this is what appears on the web interface
- Add another test with a minimal tsv for polygon (just points and label)

I need to bump version but I don't know what the current version (5.18.0) corresponds to...

### CODE REVIEW

**Comment by bernt-matthias** [comment]
> Do not expose password in command line when possible

Good idea. We should do this as well for the import tool. We Could create a config file: 

```
<configfile name="credentials">
export OMERO_USER="$__user__.extra_preferences.get('omero_account|username', $test_username)"
export OMERO_PASSWORD="$__user__.extra_preferences.get('omero_account|password', $test_password)"
</configfile>
```

and in the tool: 

```
source '$credentials' &&
omero import ...
            -u "\$OMERO_USER"
            -w "\$OMERO_PASSWORD"
```

> I need to bump version but I don't know what the current version (5.18.0) corresponds to...

That's indeed an unfortunate choice. The 1st tool that has been added here uses omero-py (5.18.0). The tools added later do not use omero-py but still use the version macro :(

**Review by bernt-matthias — APPROVED** [review]
Cool thanks for the contributions. 

Is this also fine for you @rmassei ?

File: tools/omero/omero_import.xml [hunk 1]
```diff
@@ -25,12 +25,18 @@
         omero import folder -T Dataset:name:$dataset_name
```
[the remark this answers is not in the document]
Comment (lldelisle) [reply tools/omero/omero_import.xml:26]: Thanks I realized I forgot this.

**Comment by rmassei** [comment]
Looks good to me, thanks a lot @lldelisle for the input!

**Pull request merged**